<a href="https://colab.research.google.com/github/Nataliahfk/estadistica_python/blob/main/Curso_estadistica_parte2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

***
# <font color=green size=10>CURSO DE ESTADÍSTICA - PARTE 2</font>
***

# <font color=green>1 CONOCIENDO LOS DATOS</font>
***

## <font color=green>1.1 Dataset del proyecto</font>
***

### Encuesta Nacional por Muestreo de Hogares - 2015

La **Encuesta Nacional por Muestreo de Hogares - ENMH** en Brasil anualmente investiga, de manera permanente, características generales de la población, educación, trabajo, ingresos, vivienda y otras, con frecuencia variables, de acuerdo a las necesidades de información del país, tales como las características de migración, fecundidad, nupcialidad, salud, seguridad alimentaria, entre otros temas. La recopilación de estas estadísticas constituye, durante los 49 años de realización de la investigación, un importante instrumento para la formulación, validación y evaluación de políticas orientadas al desarrollo socioeconómico y la mejora de las condiciones de vida en Brasil.

### Fuente de los Datos

Instituto Brasileño de Geografía y Estadística (IBGE) Encuesta Nacional por Muestreo de Hogares(PNAD) en el 2015

### Variables utilizadas

> ### Ingresos
> ***

Ingresos mensuales del trabajo principal para personas de 10 años o más.

> ### Edad
> ***

Edad del residente en años en la fecha de referencia .

> ### Altura (elaboración propia)
> ***

Altura del habitante en metros.

> ### UF
> ***

Unidad Federal

|Código|Descripción|
|---|---|
|11|Rondônia|
|12|Acre|
|13|Amazonas|
|14|Roraima|
|15|Pará|
|16|Amapá|
|17|Tocantins|
|21|Maranhão|
|22|Piauí|
|23|Ceará|
|24|Rio Grande do Norte|
|25|Paraíba|
|26|Pernambuco|
|27|Alagoas|
|28|Sergipe|
|29|Bahia|
|31|Minas Gerais|
|32|Espírito Santo|
|33|Rio de Janeiro|
|35|São Paulo|
|41|Paraná|
|42|Santa Catarina|
|43|Rio Grande do Sul|
|50|Mato Grosso do Sul|
|51|Mato Grosso|
|52|Goiás|
|53|Distrito Federal|

> ### Sexo
> ***

|Código|Descripción|
|---|---|
|0|Masculino|
|1|Femenino|

> ### Años de Estudio
> ***

|Código|Descripción|
|---|---|
|1|Sin educación o menos de 1 año|
|2|1 año|
|3|2 años|
|4|3 años|
|5|4 años|
|6|5 años|
|7|6 años|
|8|7 años|
|9|8 años|
|10|9 años|
|11|10 años|
|12|11 años|
|13|12 años|
|14|13 años|
|15|14 años|
|16|15 años o más|
|17|No determinado|
||No aplica|

> ### Color
> ***

|Código|Descripción|
|---|---|
|0|Indígena|
|2|Blanco|
|4|Negro|
|6|Amarillo|
|8|Pardo|
|9|Sin declarar|

#### <font color='red'>Nota</font>
***
> Se realizaron los siguientes tratamientos sobre los datos originales:
> 1. Se eliminaron los registros en los que **Ingreso** no era válido. (999 999 999 999);
> 2. Se eliminaron los registros en los que **Ingreso** eran missing;
> 3. Solo se consideraron los registros de **Personas de Referencia** de cada hogar (responsable del hogar).

### Importando pandas y leyendo el dataset del proyecto

https://pandas.pydata.org/

In [221]:
import pandas as pd

In [222]:
datos = pd.read_csv('/content/datos.csv', encoding='latin-1') # encoding='latin-1' es para no tener problemas con la ñ o alguna otra letra latina

In [223]:
datos.sample(6)

Ciudad  Sexo  Edad  Color  Años de Estudio  Ingreso    Altura
26791      29     0    67      8                9      800  1.879538
61916      43     1    49      2               10     1200  1.614926
68290      50     0    55      2                5     3000  1.637768
44824      33     1    37      8               11      980  1.833606
2634       13     0    63      8                1        0  1.761295
2929       13     1    28      8               12      600  1.752976

---
---

# <font color=green>2 DISTRIBUCIONES DE PROBABILIDAD</font>
***

## <font color=red>Problema</font>
***

En una competencia para ocupar un puesto de científico de datos, tenemos un total de **10 preguntas** de opción múltiple con **3 alternativas posibles** en cada pregunta. **Cada pregunta tiene el mismo valor.** Suponga que un candidato decide aventurarse sin haber estudiado nada. Decide hacer la prueba con los ojos vendados y adivinar todas las respuestas. Suponiendo que la prueba **vale 10 puntos y la puntuación de corte es 5**, obtenga la probabilidad de que este candidato **acierte 5 preguntas** y también la probabilidad de que el candidato **pase a la siguiente etapa del proceso de selección**.

## <font color=green>2.1 Distribución Binomial</font>
***

Un evento **binomial** se caracteriza por la posibilidad de que solo ocurran dos categorías. Estas categorías juntas representan todo el espacio muestral, siendo también mutuamente excluyentes, es decir, la ocurrencia de una implica la no ocurrencia de la otra.

En el análisis estadístico, el uso más común de la distribución binomial es para resolver problemas que involucran situaciones de **éxito** y **fracaso**.

# $$P(k)=\binom{n}{k} p^k q^{n-k}$$

Donde:

$p$ = probabilidad de éxito

$q = (1 - p)$ = probabilidad de fracaso

$n$ = número de eventos estudiados

$k$ = número de eventos deseados que tienen éxito

### Experimento Binomial

1. Realización de $n$ pruebas idénticas.

2. Las pruebas son independientes.

3. Solo son posibles dos resultados, por ejemplo: Verdadero o falso; Cara o cruz; Éxito o fracaso.

4. La probabilidad de éxito está representada por $p$ y la probabilidad de fracaso por $1-p = q$. Estas probabilidades no cambian de una prueba para otra.

### Media de la distribución binomial

El valor esperado o la media de la distribución binomial es igual al número de experimentos realizados multiplicado por la probabilidad de que ocurra el evento.

# $$\mu = n \times p$$

### Desviación estándar de la distribución binomial

La desviación estándar es la raíz quadrada del producto entre el número de experimentos, la probabilidad de éxito y la probabilidad de fracaso.

# $$\sigma = \sqrt{n \times p \times q}$$

### Importando bibliotecas
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.comb.html

In [224]:
# nueva biblioteca para poder leer las formulas matemáticas

from scipy.special import comb

### Combinaciones

El número de combinaciones de $n$ objetos, tomando $k$ cada vez, es:

# $$C_{k}^{n} = \binom{n}{k} = \frac{n!}{k!(n - k)!}$$

Donde

## $$n! = n\times(n-1)\times(n-2)\times...\times(2)\times(1)$$
## $$k! = k\times(k-1)\times(k-2)\times...\times(2)\times(1)$$

Por definición

## $$0! = 1$$

## <font color='blue'>Ejemplo: Mega Sena</font>

En una rueda de lotería Mega Sena tenemos un total de **60 números** para elegir, donde la apuesta mínima es de **seis números**. Usted que es curioso(a) decide calcular la probabilidad de ganar la Mega Sena con solo **un juego**. Para eso necesitamos saber cuántas **combinaciones de seis números se pueden formar con los 60 números disponibles**.

### $$C_{6}^{60} = \binom{60}{6} = \frac{60!}{6!(60 - 6)!}$$

In [225]:
combinaciones = comb(60,6)
combinaciones

np.float64(50063860.0)

In [226]:
probabilidad = 1/combinaciones
print('%0.15f' % probabilidad) # el resultado que me muestra si imprimo la probabilidad es: 1.997448858318156e-08 /
                               # para que no me muestre ese e-08 (8 ceros), coloco en el print '%0.15f' % probabilidad'
                               # para que me muestre con los decimales correspondientes.


0.000000019974489


EJEMPLO:

In [227]:
from scipy.special import comb

combinaciones = comb(25, 20)
probabilidad = 1 / combinaciones
print(f'Combinaciones = {combinaciones:.0f} y la Probabilidad = {probabilidad:0.15f}')

Combinaciones = 53130 y la Probabilidad = 0.000018821757952


## <font color='blue'>Ejemplo: Concurso para científico de datos</font>

En una competencia para ocupar un puesto de científico de datos, tenemos un total de **10 preguntas** de opción múltiple con **3 alternativas posibles** en cada pregunta. **Cada pregunta tiene el mismo valor.** Suponga que un candidato decide aventurarse sin haber estudiado nada. Decide tomar la prueba con los ojos vendados y adivinar todas las respuestas. Suponiendo que la prueba **vale 10 puntos y la puntuación de corte es 5**, obtenga la probabilidad de que este candidato **acierte 5 preguntas** y también la probabilidad de que el candidato **pase a la siguiente etapa del proceso de selección**.

### ¿Cuál es la cantidad de pruebas ($n$)?

In [228]:
n = 10 # n° de preguntas
n

10

### ¿Las pruebas son independientes?

Sí. La opción elegida en una pregunta no influye en la opción elegida en otra pregunta.

### ¿Sólo dos resultados son posibles en cada prueba?

Sí. El candidato tiene dos posibilidades, ACERTAR o FALLAR una pregunta.

### ¿Cuál es la probabilidad de éxito ($p$)?

In [229]:
numro_alternativas_por_pregunta = 3
p =1/numro_alternativas_por_pregunta
p

0.3333333333333333

### ¿Cuál es la probabilidad de fracaso ($q$)?

In [230]:
# probabilidad de responder incorrectamente
q = 1-p
q

0.6666666666666667

### ¿Cuál es el número total de eventos para tener éxito ($k$)?

In [231]:
# cuanto es lo minimo que tenfo que obtener para tener éxito, en este caso 5 preguntas paso
k = 5
k

5

### Solución 1

In [232]:
probabilidad = (comb(n,k) * (p ** k) * q ** (n-k)) # elevar un número a una potencia, debes usar **
print (f'{probabilidad:.8f}')

0.13656455


### Importando bibliotecas
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.binom.html

In [233]:
# para calcular binomial con binom
from scipy.stats import binom

### Solución 2

In [234]:
probabilidad = binom.pmf(k, n, p) # binom.pmf -----> Calcula probabilidad exacta de éxitos.
print(f'{probabilidad:.8f}')

0.13656455


### Obtenga la probabilidad de que el candidato apruebe

### $$P(acertar \geq 5) = P(5) + P(6) + P(7) + P(8) + P(9) + P(10)$$

**4 Formas para calcular:**

In [235]:
resultado = binom.pmf(5,n,p) + binom.pmf(6,n,p) + binom.pmf(7,n,p) + binom.pmf(8,n,p) + binom.pmf(9,n,p)
print(f'{resultado}')

0.21311114498128633


In [236]:
# paso la lista completa de los puntajes con exito / esta opción da lo mismo que la de más abajo
resultado = binom.pmf([5,6,7,8,9,10], n, p).sum()
print(f'{resultado}')

0.21312808006909476


In [237]:
# significa la probabilidad de obtener más de 4 éxitos en una distribución binomial con 𝑛 ensayos y probabilidad 𝑝 de éxito en cada ensayo
# probabilidad de obtener hasta k éxitos (es decir, 𝑋 ≤ 𝑘 ). en este caso 5
# 1 - binom.cdf(4, n, p) = probabilidad de NO estar en 𝑋 ≤ 4, lo que equivale a X > 4. ( x mayor a 4)

resultado = 1- binom.cdf(4,n,p)
print(f'{resultado}')

0.21312808006909523


In [238]:
resultado = binom.sf(4, n, p) # Probabilidad de obtener más de 4 éxitos  / es mejor usar binom.sf
print(f'{resultado}')

0.21312808006909525


La probailidad de acertar 5 preguntas es de 21.31% pase a la siguiente etapa.

**EJEMPLO:** Una moneda perfectamente equilibrada se lanza al aire cuatro veces. Usando la distribución binomial, obtenga la probabilidad de que la moneda caiga con la cruz hacia arriba dos veces

In [239]:
from scipy.stats import binom

n = 4     # número de lanzamientos
p = 0.5   # probabilidad de cruz (50% por eso es 0.5)
k = 2     # número de cruces deseados

probabilidad = binom.pmf(k,n,p)
print(f'{probabilidad:.5f}')

0.37500


**EJEMPLO:** Un dado perfectamente equilibrado se lanza hacia arriba diez veces. Usando la distribución binomial, obtenga la probabilidad de que los dados caigan con el número cinco hacia arriba al menos tres veces.

Sí es un experimento binomial porque:

- Hay un número fijo de ensayos (n = 10 lanzamientos).

- Cada ensayo tiene solo dos resultados posibles respecto al evento que nos interesa:

- Éxito = sale el número 5.

- Fracaso = no sale el número 5.

- La probabilidad de éxito es la misma en cada ensayo (p = 1/6).

- Los ensayos son independientes.

In [240]:
from scipy.stats import binom

n = 10    # número de lanzamientos
p = 1/6   # probabilidad que salga el 5 (100% / 6 posbilidades =  0.166)

print(f'{binom.sf(2, n, p):.2%}')  # el 2 de la k equivale a que al menos salga 3 veces el numero 5 (se coloca 1 número menor a lo solicitado)

22.48%


## <font color='blue'>Ejemplo: Yincana</font>

Una ciudad del interior realiza una yincana cada año para recaudar fondos para el hospital de la ciudad. En la última competencia se sabe que la **proporción de mujeres participantes fue del 60%**. **El número total de equipos, con 12 miembros, inscritos en la yincana de este año es 30**. Con la información anterior, responda: ¿Cuántos equipos deberían estar formados por **8 mujeres**?

### Solución

Datos extraidos:

- Los equipos deben tener 8 mujeres y 4 hombres, ya que el total de equipo es de 12 miembros.
- proporción de mujeres participantes fue del 60%. Por lo tanto el $40% restante fueron hombres.


In [241]:
n = 12  # cantidad de personas
p = 0.6 # 60% mujeres
k = 8   # mujeres

probabilidad = binom.pmf(k,n,p)
print(f'{probabilidad:.2%}')


21.28%


**Cuántos equipos deberían estar formados por 8 mujeres**

Datos extraidos:
- inscritos en la yincana de este año es 30.
- El número total de equipos, con 12 miembros, inscritos en la yincana de este año es 30.

###** aqui vamos a utilizar la media de distribución binomial**
- La media de una distribución binomial te dice cuántos “éxitos” esperamos en promedio si repetimos el experimento muchas veces.

In [242]:
equipos = 30 * probabilidad  # 30 ex la cantidad de experimentos
print(f'{equipos:.0f} equipos')

6 equipos


**EJEMPLO:** Suponga que la probabilidad de que una pareja tenga hijos con ojos azules es del 22%. En 50 familias, con 3 hijos cada una, ¿cuántas podemos esperar que tengan dos hijos con ojos azules?

In [243]:
# Probabilidad

n = 3         #  Cada familia tiene 3 hijos / ensayos por familia.
p = 0.22      #  Probabilidad de éxito (hijo con ojos azules) 22%
k = 2         #  Cantidad de hijos con ojos azules
familia = 50 # cantidad de familias

probabilidad = binom.pmf(k,n,p)
print(f'{probabilidad:.2%}')

11.33%


In [244]:
# La media de una distribución binomial
esperado = familia * probabilidad
print(f'{esperado:.2f} familias')

5.66 familias


---
---

## <font color=red>Problema</font>
***

Un restaurante recibe un promedio de **20 pedidos por hora**. ¿Cuál es la probabilidad de que, en un momento determinado al azar, el restaurante reciba **15 pedidos**?

## <font color=green>2.2 Distribución Poisson</font>

***

Se utiliza para describir el número de ocurrencias en un tiempo o espacio específico. Los eventos se caracterizan por la posibilidad de contar los éxitos, pero no poder contar los fracasos.

Como ejemplos de procesos donde podemos aplicar la distribución de Poisson tenemos la determinación del número de clientes que ingresan a una tienda en un momento determinado, el número de autos que llegan a un Auto-servicio de una cafetería a la hora del almuerzo, la determinación del número de accidentes registrados en un tramo de carretera, etc.

# $$P(k) = \frac{e^{-\mu}(\mu)^k}{k!}$$

Donde:

$e$ = constante cuyo valor aproximado es 2.718281828459045

$\mu$ = representa el número promedio de ocurrencias en un intervalo de tiempo o espacio dado

$k$ = número de éxitos en el intervalo deseado

### Experimento Poisson

1. La probabilidad de que ocurra es la misma en todo el intervalo observado.

2. El número de ocurrencias en un intervalo dado es independiente del número de ocurrencias en otros intervalos.

3. La probabilidad de que ocurra es la misma en intervalos de igual longitud.

### Media de la distribución Poisson

# $$\mu$$

### Desviación estándar de la distribución Poisson

# $$\sigma = \sqrt{\mu}$$

<br><br>**De forma simple:**

La Distribución de Poisson sirve para modelar la probabilidad de que ocurra un número de eventos en un intervalo fijo (tiempo, espacio, distancia, etc.), cuando:

- Los eventos ocurren de manera independiente.
- La tasa de ocurrencia promedio (𝜆) es constante.
- Los eventos no pueden ocurrir al mismo tiempo exacto (uno a uno).

**Fórmula**
- k = número de eventos que queremos
- λ (lambda) = número promedio de eventos en el intervalo
- e ≈ 2.718 (constante matemática)

🔹 Ejemplo simple
Supongamos que un café recibe en promedio 3 clientes por hora (𝜆=3), ¿cuál es la probabilidad de que lleguen exactamente 5 clientes en la próxima hora?<br>

P(X=5)= ( 3 ** 5 .e **−3 ) / 5! Resultado= ≈0.1008

✅ Hay 10,08% de probabilidad de que lleguen 5 clientes.
<br><br>

💡 En palabras simples:

Poisson te dice la probabilidad de “X veces que ocurra algo” cuando sabes el promedio de ocurrencias.

### Importando bibliotecas

http://www.numpy.org/

In [245]:
import numpy as np
from scipy.special import factorial

In [246]:
np.e   # .e de la formula anterior

2.718281828459045

## <font color='blue'>Ejemplo: Delivery</font>

Un restaurante recibe un promedio de **20 pedidos por hora**. ¿Cuál es la probabilidad de que, en un momento determinado al azar, el restaurante reciba **15 pedidos**?

### ¿Cuál es el número promedio de ocurrencias por hora ($\mu$)?

In [247]:
media = 20  # se refiere a los pedidos en determinado tiempo

### ¿Qué número de ocurrencias queremos obtener en el período ($k$)?

In [248]:
k = 15   # ¿Cuál es la probabilidad de que el restaurante reciba 15 pedidos?

### Solución 1

In [249]:
probabilidad = (media ** k * np.e ** (-media)) / factorial(k) # esto es la formula
print(f'{probabilidad:.8f}')

0.05164885


### Importando bibliotecas

https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.poisson.html

### Solución 2

In [250]:
from scipy.stats import poisson

In [251]:
probabilidad = poisson.pmf(k, media)
print(f'{probabilidad:.2%}')

5.16%


**EJEMPLO:** El número media de clientes que ingresan a una panadería por hora es 20. Obtenga la probabilidad de que, en la próxima hora, ingresen exactamente 25 clientes.

In [252]:
media = 20
k = 25

probabilidad = poisson.pmf(k, media)
print(f'{probabilidad:.2%}')

4.46%


---
---

## <font color=red>Problema</font>
***

En un estudio sobre la altura de los residentes de una ciudad, se encontró que el conjunto de datos sigue una **distribución aproximadamente normal**, con **media 1.70** y **desviación estándar de 0.1**. Con esta información, obtenga el siguiente conjunto de probabilidades:

> **A.** probabilidad de que una persona, seleccionada al azar, sea menor de 1.80 metros.

> **B.** probabilidad de que una persona, seleccionada al azar, tenga entre 1.60 metros y 1.80 metros.

> **C.** probabilidad de que una persona, seleccionada al azar, tenga más de 1.90 metros.


## <font color=green>2.3 Distribución Normal</font>
***

La distribución normal es una de las más utilizadas en estadística. Es una distribución continua, donde la distribución de frecuencias de una variable cuantitativa tiene forma de campana y es simétrica en relación con su media.

<img src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img001.png' width='500px'>

### Características importantes

1. Es simétrica alrededor de la media; los datos se concentran alrededor de la media.

2. El área bajo la curva corresponde a la proporción 1 o 100%;

3. Las medidas de tendencia central (media, mediana y moda) tienen el mismo valor;

4. Los extremos de la curva tienden al infinito en ambas direcciones y, teóricamente, nunca tocan el eje $x$;

5. La desviación estándar define la planitud y el ancho de la distribución. Las curvas más anchas y planas tienen valores de desviación estándar más altos;

6. La distribución se define por su media y desviación estándar;

7. La probabilidad siempre será igual al área bajo la curva, delimitada por los límites superior e inferior.

# $$f(x) = \frac{1}{\sqrt{2\pi\sigma}}e^{-\frac{1}{2}\left(\frac{x-\mu}{\sigma}\right)^2}$$

Donde:

$x$ = variable normal

$\sigma$ = desviación estándar

$\mu$ = media

La probabilidad se obtiene del área bajo la curva, delimitada por los límites superior e inferior especificados. Se puede ver un ejemplo en la figura siguiente.

<img src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img002.png' width='500px'>


Para obtener el área anterior, simplemente calcule la integral de la función para los intervalos determinados. Según la siguiente ecuación:

# $$P(L_i<x<L_s) = \int_{L_i}^{L_s}\frac{1}{\sqrt{2\pi\sigma}}e^{-\frac{1}{2}\left(\frac{x-\mu}{\sigma}\right)^2}$$

Donde:

$x$ = variable normal

$\sigma$ = desviación estándar

$\mu$ = media

$L_i$ = límite inferior

$L_s$ = límite superior

### Tablas estandarizadas

Se crearon tablas estandarizadas para facilitar la obtención de los valores de las áreas bajo la curva normal y eliminar la necesidad de resolver integrales definidas.

Para ver los valores en una tabla estandarizada, simplemente transforme nuestra variable en una variable estandarizada $Z$.

Esta variable $Z$ representa la desviación en las desviaciones estándar de un valor de la variable original en relación con la media.

# $$Z = \frac{x-\mu}{\sigma}$$

Donde:

$x$ = variable normal con media $\mu$ y desviación estándar $\sigma$

$\sigma$ = desviación estándar

$\mu$ = media

### Construyendo la tabla normal estandarizada
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html

In [253]:
import pandas as pd
import numpy as np
from scipy.stats import norm

tabla_normal_estandarizada = pd.DataFrame(
    [],
    index=["{0:0.2f}".format(i / 100) for i in range(0, 400, 10)],
    columns = ["{0:0.2f}".format(i / 100) for i in range(0, 10)])

for index in tabla_normal_estandarizada.index:
    for column in tabla_normal_estandarizada.columns:
        Z = np.round(float(index) + float(column), 2)
        tabla_normal_estandarizada.loc[index, column] = "{0:0.4f}".format(norm.cdf(Z))

tabla_normal_estandarizada.rename_axis('Z', axis = 'columns', inplace = True)

tabla_normal_estandarizada

Z       0.00    0.01    0.02    0.03    0.04    0.05    0.06    0.07    0.08  \
0.00  0.5000  0.5040  0.5080  0.5120  0.5160  0.5199  0.5239  0.5279  0.5319   
0.10  0.5398  0.5438  0.5478  0.5517  0.5557  0.5596  0.5636  0.5675  0.5714   
0.20  0.5793  0.5832  0.5871  0.5910  0.5948  0.5987  0.6026  0.6064  0.6103   
0.30  0.6179  0.6217  0.6255  0.6293  0.6331  0.6368  0.6406  0.6443  0.6480   
0.40  0.6554  0.6591  0.6628  0.6664  0.6700  0.6736  0.6772  0.6808  0.6844   
0.50  0.6915  0.6950  0.6985  0.7019  0.7054  0.7088  0.7123  0.7157  0.7190   
0.60  0.7257  0.7291  0.7324  0.7357  0.7389  0.7422  0.7454  0.7486  0.7517   
0.70  0.7580  0.7611  0.7642  0.7673  0.7704  0.7734  0.7764  0.7794  0.7823   
0.80  0.7881  0.7910  0.7939  0.7967  0.7995  0.8023  0.8051  0.8078  0.8106   
0.90  0.8159  0.8186  0.8212  0.8238  0.8264  0.8289  0.8315  0.8340  0.8365   
1.00  0.8413  0.8438  0.8461  0.8485  0.8508  0.8531  0.8554  0.8577  0.8599   
1.10  0.8643  0.8665  0.8686  0.8708  0.8729  0.8749  0.8770  0.8790  0.8810   
1.20  0.8849  0.8869  0.8888  0.8907  0.8925  0.8944  0.8962  0.8980  0.8997   
1.30  0.9032  0.9049  0.9066  0.9082  0.9099  0.9115  0.9131  0.9147  0.9162   
1.40  0.9192  0.9207  0.9222  0.9236  0.9251  0.9265  0.9279  0.9292  0.9306   
1.50  0.9332  0.9345  0.9357  0.9370  0.9382  0.9394  0.9406  0.9418  0.9429   
1.60  0.9452  0.9463  0.9474  0.9484  0.9495  0.9505  0.9515  0.9525  0.9535   
1.70  0.9554  0.9564  0.9573  0.9582  0.9591  0.9599  0.9608  0.9616  0.9625   
1.80  0.9641  0.9649  0.9656  0.9664  0.9671  0.9678  0.9686  0.9693  0.9699   
1.90  0.9713  0.9719  0.9726  0.9732  0.9738  0.9744  0.9750  0.9756  0.9761   
2.00  0.9772  0.9778  0.9783  0.9788  0.9793  0.9798  0.9803  0.9808  0.9812   
2.10  0.9821  0.9826  0.9830  0.9834  0.9838  0.9842  0.9846  0.9850  0.9854   
2.20  0.9861  0.9864  0.9868  0.9871  0.9875  0.9878  0.9881  0.9884  0.9887   
2.30  0.9893  0.9896  0.9898  0.9901  0.9904  0.9906  0.9909  0.9911  0.9913   
2.40  0.9918  0.9920  0.9922  0.9925  0.9927  0.9929  0.9931  0.9932  0.9934   
2.50  0.9938  0.9940  0.9941  0.9943  0.9945  0.9946  0.9948  0.9949  0.9951   
2.60  0.9953  0.9955  0.9956  0.9957  0.9959  0.9960  0.9961  0.9962  0.9963   
2.70  0.9965  0.9966  0.9967  0.9968  0.9969  0.9970  0.9971  0.9972  0.9973   
2.80  0.9974  0.9975  0.9976  0.9977  0.9977  0.9978  0.9979  0.9979  0.9980   
2.90  0.9981  0.9982  0.9982  0.9983  0.9984  0.9984  0.9985  0.9985  0.9986   
3.00  0.9987  0.9987  0.9987  0.9988  0.9988  0.9989  0.9989  0.9989  0.9990   
3.10  0.9990  0.9991  0.9991  0.9991  0.9992  0.9992  0.9992  0.9992  0.9993   
3.20  0.9993  0.9993  0.9994  0.9994  0.9994  0.9994  0.9994  0.9995  0.9995   
3.30  0.9995  0.9995  0.9995  0.9996  0.9996  0.9996  0.9996  0.9996  0.9996   
3.40  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997   
3.50  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998   
3.60  0.9998  0.9998  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999   
3.70  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999   
3.80  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999   
3.90  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000   

Z       0.09  
0.00  0.5359  
0.10  0.5753  
0.20  0.6141  
0.30  0.6517  
0.40  0.6879  
0.50  0.7224  
0.60  0.7549  
0.70  0.7852  
0.80  0.8133  
0.90  0.8389  
1.00  0.8621  
1.10  0.8830  
1.20  0.9015  
1.30  0.9177  
1.40  0.9319  
1.50  0.9441  
1.60  0.9545  
1.70  0.9633  
1.80  0.9706  
1.90  0.9767  
2.00  0.9817  
2.10  0.9857  
2.20  0.9890  
2.30  0.9916  
2.40  0.9936  
2.50  0.9952  
2.60  0.9964  
2.70  0.9974  
2.80  0.9981  
2.90  0.9986  
3.00  0.9990  
3.10  0.9993  
3.20  0.9995  
3.30  0.9997  
3.40  0.9998  
3.50  0.9998  
3.60  0.9999  
3.70  0.9999  
3.80  0.9999  
3.90  1.0000

<img src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img003.png' width='250px'>

La tabla anterior proporciona el área bajo la curva entre $-\infty$ y $Z$ desviaciones estándar por encima de la media. Recordando que dado que estos son valores estandarizados, tenemos $\mu = 0$.

## <font color='blue'>Ejemplo: ¿Cuánto mides?</font>

En un estudio sobre la altura de los residentes de una ciudad, se encontró que el conjunto de datos sigue una **distribución aproximadamente normal**, con **media 1.70** y **desviación estándar de 0.1**. Con esta información, obtenga el siguiente conjunto de probabilidades:

> **A.** probabilidad de que una persona, seleccionada al azar, mida menos que 1.80 metros.

> **B.** probabilidad de que una persona, seleccionada al azar, mida entre 1.60 metros y 1.80 metros.

> **C.** probabilidad de que una persona, seleccionada al azar, mida más de 1.90 metros.

### Problema A - Identificación del área bajo la curva

> **A.** probabilidad de que una persona, seleccionada al azar, mida menos que 1.80 metros.

<img style='float: left' src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img004.png' width='350px'>


### Obtener la variable estandarizada $Z$

In [254]:
# Construir variable estandarizada

media = 1.70
desviacion_estandar = 0.1
x = 1.80

z = (x - media) / desviacion_estandar
print(z)

1.0000000000000009


### Solución 1 - Utilizando la tabla

En este caso nos da un valor 1. Luego debemos buscar en la tabla normal estandarizada el valor 1.00 en las filas y en la columna 0.0
en el caso que tuviesemos que buscar ej: el valor 15 , buscamos en la fila el 10 y en la columna el 0.5 y la coordenada que de ese es el valor.

En este caso 1 el valor en la tabla es 0.8413

In [255]:
probabilidad = 0.8413 # equivale a 84,13 %
probabilidad

0.8413

### Solución 2 - Utilizando Scipy

In [256]:
from scipy.stats import norm

probabilidad = norm.cdf(z) # calcula la probabilidad acumulada hasta un valor z en una distribución normal estándar
                           # (media 0 y desviación estándar 1)
                           # norm.cdf(z) te dice “qué tan probable es obtener un valor menor o igual a z” en la distribución normal.

print(f'{probabilidad:.2%}')

84.13%


**EJEMPLO:** La aplicación de una prueba Estadística en una competencia presentó un conjunto de notas normalmente distribuidas. Se encontró que el conjunto de notas tenía una media de 70 y una desviación estándar de 5 puntos.

¿Qué probabilidad hay de que un estudiante, seleccionado al azar, obtenga una puntuación inferior a 85 puntos?

In [257]:
media = 70
desviacion_estandar = 5
x = 85

z = (x - media) / desviacion_estandar
print(z)  # este resultado lo debo ir a buscar a la tabla normal estandarizada / fila 3 columna 0 / sino con la formula norm.cdf

3.0


In [258]:
probabilidad = 0.9987

In [259]:
from scipy.stats import norm

probabilidad = norm.cdf(z)
print(f'{probabilidad:.2%}')

99.87%


### Problema B - Identificación del área bajo la curva

> **B.** probabilidad de que una persona, seleccionada al azar, mida entre 1.60 metros y 1.80 metros.


<img style='float: left' src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img005.png' width='350px'>

### Obtener la variable estandarizada $Z$

###**Solución** 1 - Utilizando la tabla

In [260]:
media = 1.70
desviacion_estandar = 0.1

z = (1.70 - media) / desviacion_estandar
print(z)

0.0


In [261]:
probabilidad = 0.5000 # 50%
probabilidad

0.5

### Solución 2 - Utilizando Scipy

In [262]:
from scipy.stats import norm

probabilidad = norm.cdf(z)
print(f'{probabilidad:.2%}')

50.00%


### Solución 1 - Utilizando la tabla

In [263]:
media = 1.70
desviacion_estandar = 0.1

z = (1.80 - media) / desviacion_estandar
print(z)

1.0000000000000009


In [264]:
probabilidad = 0.8413
probabilidad

0.8413

In [265]:
# estoy calculando la probabilidad del 1.80 - la probabilidad del 1.70 mts y calculamos el area que es el doble de lo que esta pintado en
# el gráfico por eso multiplico *2
probabilidad = (0.8413 - 0.5000) * 2
probabilidad

0.6826000000000001

### Solución 2 - Utilizando Scipy

In [266]:
z_superior = (1.80 - media) / desviacion_estandar
print(z)

1.0000000000000009


In [267]:
probabilidad = norm.cdf(z_superior) - (1- norm.cdf(z_superior))  # (1- norm.cdf(z_superior)) esto equivale al area no sombreada
print(f'{probabilidad:.2%}')

68.27%


In [268]:
z_inferior = (1.60 - media) / desviacion_estandar
print(z)

1.0000000000000009


In [269]:
probabilidad = norm.cdf(z_superior) - norm.cdf(z_inferior)
print(f'{probabilidad:.2%}')

68.27%


####Otra forma para resolverlo:

1️⃣ Entender qué significa el problema
Queremos:

P(1.60≤X≤1.80)

donde
X sigue una distribución normal con:
- Media (𝜇) = 1.70 m
- Desviación estándar (σ) = 0.1 m

2️⃣ Convertir a puntuaciones Z

La fórmula del Z-score es:

       z= X−μ / σ


Para 1.60 m:
z  = 1.60−1.70 / 0.1 = −0.10 / 0.1 = −1
​<br>
Para 1.80 m:
z  = 1.80−1.70 / 0.1 = 0.10 / 0.1 = 1

3️⃣ Buscar las probabilidades acumuladas (CDF)​<br>
Usamos tabla Z o norm.cdf:

- P(Z≤1)≈0.8413
- 𝑃(𝑍≤−1)≈0.1587

<br>

📌 Esto significa:

- Un Z = 1 cubre el 84,13% de la distribución desde el extremo izquierdo hasta ese punto.

- Un Z = -1 cubre el 15,87% desde el extremo izquierdo hasta ese punto.
<br>

4️⃣ Restar para obtener la probabilidad en el rango

Queremos el área entre Z = -1 y Z = 1:

        P(−1≤Z≤1)=0.8413−0.1587=0.6826

✅ Resultado: 0.6826 → un 68,26% de las personas miden entre 1.60 m y 1.80 m.


          from scipy.stats import norm
          valor = norm.cdf(-1)  # P(Z <= -1)
          print(valor)

          0.15865525393145707


**EJEMPLO:**La facturación diaria de un controlador de aplicación sigue una distribución aproximadamente normal, con un media de $ 300,00 y una desviación estándar igual a $ 50,00. Obtenga las probabilidades de que, en un día aleatorio, el conductor gane:

1) Entre $ 250,00 y $ 350,00


In [270]:
media = 300
desviacion_estandar = 50

z1 =(250 - media)/ desviacion_estandar
print(z1)

-1.0


In [271]:
media = 300
desviacion_estandar = 50

z2 =(350 - media)/ desviacion_estandar
print(z)

1.0000000000000009


In [272]:
prob_acumulada1 = norm.cdf(z1)
print(prob_acumulada1)

0.15865525393145707


In [273]:
prob_acumulada2 = norm.cdf(z2)
print(prob_acumulada2)

0.8413447460685429


In [274]:
probabilidad = prob_acumulada2 - prob_acumulada1
print(f'{probabilidad:.2%}')

68.27%


2) Entre $ 400,00 y $ 500,00

In [275]:
media = 300
desviacion_estandar = 50

z1 = (400 - media)/ desviacion_estandar
print(z1)

2.0


In [276]:
media = 300
desviacion_estandar = 50

z2 = (500 - media)/ desviacion_estandar
print(z2)

4.0


In [277]:
prob_acumulada1 = norm.cdf(z1)
print(prob_acumulada1)

0.9772498680518208


In [278]:
prob_acumulada2 = norm.cdf(z2)
print(prob_acumulada2)

0.9999683287581669


In [279]:
probabilidad = prob_acumulada2 - prob_acumulada1
print(f'{probabilidad:.2%}')

2.27%


In [280]:
from scipy.stats import norm

media = 300
desviacion_estandar = 50
Z_inferior = (250 - media) / desviacion_estandar
Z_superior = (350 - media) / desviacion_estandar

probabilidad = norm.cdf(Z_superior) - norm.cdf(Z_inferior)
print(f'{probabilidad:.2%}')

68.27%


In [281]:
from scipy.stats import norm

media = 300
desviacion_estandar = 50
Z_inferior = (400 - media) / desviacion_estandar
Z_superior = (500 - media) / desviacion_estandar

probabilidad = norm.cdf(Z_superior) - norm.cdf(Z_inferior)
print(f'{probabilidad:.2%}')

2.27%


### Problema C - Identificación del área bajo la curva

> **C.** probabilidad de que una persona, seleccionada al azar, mida más de 1.90 metros.

<img style='float: left' src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img006.png' width='350px'>

### Obtener la variable estandarizada $Z$

In [282]:
media = 1.70
desviacion_estandar = 0.1

z = (1.90 - media) / desviacion_estandar
print(z)

1.9999999999999996


### Solución 1 - Utilizando la tabla

In [283]:
probabilidad = 1 - 0.9767   # valor que me aparece en la tabla coordenadas 1.90 + 0.09 = 0.9767  y el 1 es el área que resta
probabilidad

0.023299999999999987

### Solución 2 - Utilizando Scipy

In [286]:
probabilidad_acumulada = norm.cdf(z)
print(probabilidad_acumulada)

0.9772498680518208


In [287]:
probabilidad = 1- probabilidad_acumulada
print(f'{probabilidad:.2%}')

2.28%


In [284]:
probabilidad = 1 - norm.cdf(z)
print(f'{probabilidad:.2%}')

2.28%


In [285]:
probabilidad = norm.cdf(-z)
print(f'{probabilidad:.2%}')

2.28%


**EJEMPLOS:** Una empresa verificó que las lámparas incandescentes XPTO tienen una vida útil normalmente distribuida, con un media igual a 720 días y una desviación estándar igual a 30 días. Calcula la probabilidad de que una lámpara, elegida al azar, dure:

1) Entre 650 y 750 días

2) Más de 800 días

3) Menos de 700 días

In [290]:
media = 720
desviacion_estandar = 30

z1 = (650 - media) / desviacion_estandar
z2 = (750 - media) / desviacion_estandar

probabilidad = norm.cdf(z2) - norm.cdf(z1)
print(f'{probabilidad:.2%}')

83.15%


In [293]:
media = 720
desviacion_estandar = 30

z3 = (800 - media) / desviacion_estandar
probabilidad = 1 - norm.cdf(z3)
print(f'{probabilidad:.2%}')

0.38%


In [294]:
media = 720
desviacion_estandar = 30

z4 = (700 - media) / desviacion_estandar
probabilidad = norm.cdf(-z4)
print(f'{probabilidad:.2%}')

74.75%


# <font color=green>3 MUESTREO</font>
***

## <font color=green>3.1 Población y Muestra</font>
***

### Población
Conjunto de todos los elementos de interés de un estudio. Varios elementos pueden componer una población, por ejemplo: personas, edades, alturas, automóviles, etc.

En cuanto al tamaño, las poblaciones pueden ser limitadas (poblaciones finitas) o ilimitadas (poblaciones infinitas).

### Poblaciones finitas

Permiten el recuento de sus elementos. Los ejemplos incluyen el número de empleados en una empresa, el número de estudiantes en una escuela, etc.

### Poblaciones infinitas

No es posible contar sus elementos. Como ejemplos tenemos el número de porciones que se pueden extraer del agua del mar para su análisis, la temperatura medida en cada punto de un territorio, etc.

<font color=red>Cuando se pueden contar los elementos de una población, pero presentando una cantidad muy grande, se supone que la población es infinita.</font>

### Muestra
Subconjunto representativo de la población.

Los atributos numéricos de una población, como su media, varianza y desviación estándar, se conocen como **parámetros**. El enfoque principal de la inferencia estadística es precisamente generar estimaciones y probar hipótesis sobre parámetros poblacionales utilizando información de la muestra.

## <font color=green>3.2 ¿Cuándo usar una muestra?</font>
***

### Poblaciones infinitas

El estudio nunca llegaría a su fin. No es posible investigar todos los elementos de la población.

### Pruebas destructivas

Estudios donde los elementos evaluados son totalmente consumidos o destruidos. Ejemplo: pruebas de vida útil, pruebas de seguridad en accidentes de tráfico.

### Resultados rápidos

Investigaciones que necesitan más agilidad en la difusión. Ejemplo: encuestas de opinión, encuestas sobre problemas de salud pública.

### Altos precios

Cuando la población es finita pero muy grande, el costo de un censo puede hacer inviable el proceso.

## <font color=green>3.3 Muestreo Aleatorio Simple</font>
***

Es una de las principales formas de extraer una muestra de una población. El requisito fundamental de este tipo de enfoque es que cada elemento de la población tenga las mismas posibilidades de ser seleccionado para formar parte de la muestra.

## <font color=green>3.4 Muestreo Estratificado</font>
***

Es una mejora del proceso de muestreo aleatorio simple. En este método, se propone dividir la población en subgrupos de elementos con características similares, es decir, grupos más homogéneos. Con estos subgrupos separados, la técnica de muestreo aleatorio simple dentro de cada subgrupo se aplica individualmente.

## <font color=green>3.5 Muestreo por Conglomerados</font>
***

También tiene como objetivo mejorar el criterio de muestreo aleatorio simple. En el muestreo por conglomerados también se crean subgrupos, pero no serán homogéneos como en el muestreo estratificado. En el muestreo por conglomerados los subgrupos serán heterogéneos, donde luego se aplicará un muestreo aleatorio simple o estratificado.

Un ejemplo muy común de la aplicación de este tipo de técnica es la división de la población en grupos territoriales, donde los elementos investigados tendrán características bastante variadas.

# <font color=green>4 ESTIMACIÓN</font>
***

## <font color='red'>Problema </font>

Suponga que los pesos de las bolsas de arroz de una industria alimentaria se distribuyen aproximadamente como una distribución normal **desviación estándar igual a 150 g**. Se seleccionó una **muestra aleatoria de 20 bolsas** de un lote específico, se obtuvo un **peso promedio de 5.050 g**. Construya un **intervalo de confianza para la media de la población** asumiendo un **nivel de significancia del 5%**.

---

Es la forma de hacer suposiciones generalizadas sobre los parámetros de una población a partir de información de una muestra.

- **Los parámetros** son los atributos numéricos de una población, como la media, la desviación estándar, etc.

- **Estimación** es el valor obtenido para un parámetro dado a partir de los datos de una muestra de la población.

## <font color=green>4.1 Teorema central del límite</font>
***

> El **Teorema Central del Límite** establece que, con el aumento del tamaño de la muestra, la distribución de las medias muestrales se aproxima a una distribución normal con media igual a la media de la población y una desviación estándar igual a la desviación estándar de la variable original dividida por la raíz cuadrada del tamaño de la muestra. Este hecho está asegurado para $n$ mayores o iguales a 30.

# $$\sigma_\bar{x} = \frac{\sigma}{\sqrt{n}}$$

La desviación estándar de las medias muestrales se conoce como **error estándar de la media**

### Entendiendo el Teorema Central del Límite

> El **Teorema Central del Límite** establece que, con el aumento del tamaño de la muestra, la distribución de las medias muestrales se aproxima a una distribución normal con media igual a la media de la población y una desviación estándar igual a la desviación estándar de la variable original dividida por la raíz cuadrada del tamaño de la muestra. Este hecho está asegurado para $n$ mayores o iguales a 30.

> El **Teorema Central del Límite** establece que, con el aumento del tamaño de la muestra, la distribución de las medias muestrales se aproxima a una distribución normal con media igual a la media de la población y una desviación estándar igual a la desviación estándar de la variable original dividida por la raíz cuadrada del tamaño de la muestra. Este hecho está asegurado para $n$ mayores o iguales a 30.

> El **Teorema Central del Límite** establece que, con el aumento del tamaño de la muestra, la distribución de las medias muestrales se aproxima a una distribución normal con media igual a la media de la población y una desviación estándar igual a la desviación estándar de la variable original dividida por la raíz cuadrada del tamaño de la muestra. Este hecho está asegurado para $n$ mayores o iguales a 30.

# $$\sigma_\bar{x} = \frac{\sigma}{\sqrt{n}}$$

## <font color=green>4.2 Los niveles de confianza y significación</font>
***

El **nivel de confianza** ($1 - \alpha$) representa la probabilidad de que la estimación sea correcta. Además, el **nivel de significancia** ($\alpha$) expresa la probabilidad de error de la estimación.

El **nivel de confianza** representa el grado de confiabilidad del resultado estimado dentro de un cierto rango. Cuando establecemos un **nivel de confianza del 95%** en una encuesta, por ejemplo, estamos asumiendo que existe una probabilidad del 95% de que los resultados de la encuesta representen bien la realidad, es decir, sean correctos.

El **nivel de confianza** de una estimación se puede obtener del área bajo la curva normal como se muestra en la figura siguiente.

<img src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img007.png' width='450px'>


## <font color=green>4.3 Error inferencial</font>
***

El **error inferencial** se define por la **desviación estándar de las medias muestrales** $\sigma_\bar{x}$ y por el **nivel de confianza** determinado para el proceso.

# $$e = z \frac{\sigma}{\sqrt{n}}$$

## <font color=green>4.4 Intervalos de confianza</font>
***

### Intervalo de confianza para la media poblacional

#### Con desviación estándar poblacional conocida

## $$\mu = \bar{x} \pm z\frac{\sigma}{\sqrt{n}}$$

#### Con desviación estándar de población desconocida

## $$\mu = \bar{x} \pm z\frac{s}{\sqrt{n}}$$

## <font color='blue'>Ejemplo: </font>

Suponga que los pesos de las bolsas de arroz de una industria alimentaria se distribuyen aproximadamente como una población normal con **desviación estándar igual a 150 g**. Se seleccionó una **muestra aleatoria de 20 bolsas** de un lote específico, se obtuvo un **peso promedio de 5.050 g**. Construya un intervalo de confianza para el **promedio de la población** asumiendo un **nivel de significancia del 5%**.

### Media muestral

### Nivel de significancia ($\alpha$)

### Nivel de confianza ($1 - \alpha$)

### Obteniendo $z$

### Obteniendo $z$

<img src='https://caelum-online-public.s3.amazonaws.com/ESP-1764-estadistica-con-python-probabilidad-y-muestreo/img008.png' width='450px'>

### Valores de $z$ para los niveles de confianza más utilizados

|Nivel de<br>confianza|Valor del área bajo<br>la curva normal| $z$ |
|:----------------:|:---------------------------------:|:---:|
|90%               |0.95                               |1.645|
|95%               |0.975                              |1.96 |
|99%               |0.995                              |2.575|

### Obteniendo $\sigma_\bar{x}$

### Obteniendo $e$

### Solución 1 - Calculando el intervalo de confianza para la media

### Solución 2 - Calculando el intervalo de confianza para la media

# <font color=green>5 CÁLCULO DEL TAMAÑO DE LA MUESTRA</font>
***

## <font color='red'>Problema </font>

Estamos estudiando los ingresos mensuales de los jefes de hogar con ingresos de hasta R$\$$ 5.000,00 en Brasil. Nuestro supervisor determinó que el **error máximo en relación a la media es de R$\$$ 10,00**. Sabemos que la **desviación estándar poblacional** de este grupo de trabajadores es **R$\$$ 1.082,79**. Para un **nivel de confianza del 95%**, ¿cuál debería ser el tamaño de la muestra de nuestro estudio?

## <font color=green>5.1 Variables cuantitativas y población infinita</font>
***

# $$e = z \frac{\sigma}{\sqrt{n}}$$

#### Con desviación estándar conocida

## $$n = \left(z\frac{\sigma}{e}\right)^2$$

#### Con desviación estándar desconocida

## $$n = \left(z\frac{s}{e}\right)^2$$

Donde:

$z$ = variable normal estandarizada

$\sigma$ = desviación estándar poblacional

$s$ = desviación estándar de la muestra

$e$ = error inferencial

### <font color='red'>Comentarios</font>

1. La desviación estándar ($\sigma$ o $s$) y el error ($e$) deben estar en la misma unidad de medida.

2. Cuando el error ($e$) se representa en términos porcentuales, debe interpretarse como un porcentaje relacionado con el promedio.

## <font color='blue'>Ejemplo: Rendimiento medio</font>

Estamos estudiando los ingresos mensuales de los jefes de hogar. Nuestro supervisor determinó que el **error máximo en relación a la media es de $\$$ 100,00** pesos. Sabemos que la **desviación estándar poblacional** de este grupo de trabajadores es **$\$$ 3.323,39** pesos. Para un **nivel de confianza del 95%**, ¿cuál debería ser el tamaño de la muestra de nuestro estudio?

### Obteniendo $\sigma$

### Obteniendo $e$

### Obteniendo $n$

---
---

## <font color='red'>Problema</font>

En un lote de **10 000 latas** de gaseosa, se realizó una muestra aleatoria simple de **100 latas** y se obtuvo que la **desviación estándar muestral del contenido de las latas es igual a 12 ml**. El fabricante estipula un **error máximo en la media poblacional de solo 5 ml**. Para garantizar un **nivel de confianza del 95%**, ¿qué tamaño de muestra se debe seleccionar para este estudio?

## <font color=green>5.2 Variables cuantitativas y población finita</font>
***

#### Con desviación estándar conocida

## $$n = \frac{z^2 \sigma^2 N}{z^2 \sigma^2 + e^2(N-1)}$$

#### Con desviación estándar desconocida

## $$n = \frac{z^2 s^2 N}{z^2 s^2 + e^2(N-1)}$$

Donde:

$N$ = tamaño de la población

$z$ = variable normal estandarizada

$\sigma$ = desviación estándar poblacional

$s$ = desviación estándar muestral

$e$ = error inferencial

## <font color='blue'>Ejemplo: Industria de refrescos</font>

En un lote de **10 000 latas** de gaseosa, se realizó una muestra aleatoria simple de **100 latas** y se obtuvo que la **desviación estándar muestral del contenido de las latas es igual a 12 ml**. El fabricante estipula un **error máximo en la media poblacional de solo 5 ml**. Para garantizar un **nivel de confianza del 95%**, ¿qué tamaño de muestra se debe seleccionar para este estudio?

### Obteniendo $N$

### Obteniendo $z$

### Obteniendo $s$

### Obteniendo $e$

### Obteniendo $n$

## $$n = \frac{z^2 s^2 N}{z^2 s^2 + e^2(N-1)}$$

# <font color=green>6 FIJANDO EL CONTENIDO</font>
***

## <font color='blue'>Ejemplo: Rendimiento medio</font>

Estamos estudiando **los ingresos mensuales de los jefes de hogar con ingresos de hasta R$\$$ 5.000,00 en Brasil**. Nuestro supervisor determinó que el **error máximo en relación a la media es de R$\$$ 10,00**. Sabemos que la **desviación estándar poblacional** de este grupo de trabajadores es **R$\$$ 1.082,79** y que la **media poblacional** es **R$\$$ 1.426,54**. Para un **nivel de confianza del 95%**, ¿cuál debería ser el tamaño de la muestra de nuestro estudio? ¿Cuál es el intervalo de confianza para la media considerando el tamaño de muestra obtenido?

### Construyendo el conjunto de datos según lo especificado por el problema

### Calcular el tamaño de la muestra

### Calcular el intervalo de confianza para la media

### Realización de una prueba gráfica